### ISSUE TITLE
Resizable SplitPageLayout.Pane is not keyboard accessible

### ISSUE THREAD

**Issue by jdrush89** [primer/react#2522]
### Description

The resizer on the SplitPageLayout.Pane should be focusable and able to be adjusted using the arrow keys.

### Steps to reproduce

1. Go to https://primer.style/react/SplitPageLayout
2. Try to focus the resizer in the resizable example using tab
3. See that the resizer is not in the tab order and cannot be change with the keyboard.

### Version

v35.14.0

### Browser

Chrome
